.github/dependabot.yml

```diff
@@ -3,9 +3,19 @@ updates:
   - package-ecosystem: gomod
     directory: "/"
     schedule:
-      interval: 'daily'
-      time: '00:00'
-      timezone: 'America/Los_Angeles' # 12am Pacific Time
+      interval: 'weekly'
     labels:
       - 'dependabot'
     open-pull-requests-limit: 5
+    groups:
+      # We only want to automate our dependencies that are not as big of moving pieces
+      minor-dependencies:
+        patterns:
+          - "*"
+        # Skip docker and k8 dependencies. We want to manually keep an eye on these
+        exclude-patterns:
+          - "*/docker/*"
```

Comment:
Since the order looks arbitrary here, can we group `docker` with `moby`, and put `opencontainers` after?

Otherwise, LGTM :+1:

Reply:
I'm going with that you meant groups within the exclude patterns and not actual dependabot groups